packages/eslint-plugin/tests/rules/no-unnecessary-type-arguments.test.ts

```diff
@@ -53,6 +53,7 @@ ruleTester.run('no-unnecessary-type-arguments', rule, {
     `declare const C: unknown;
       class D<TD = number> extends C { }`,
     `let a: A<number>`,
+    `type Foo<T> = import('foo').Foo<T>;`,
```

Comment:
could you please also add a test for a new expression to help prevent regressions?

Reply:
Oke ;)